.github/workflows/waf-sync-lambda-deploy-gf.yml

```diff
@@ -60,10 +60,10 @@ jobs:
           AWS_REGION: ${{ vars.AWS_REGION }}
         with:
           params: |
-            S3_BUCKET=/bcda/prod/bcda-prod-api-waf-sync-bucket
+            S3_BUCKET=/bcda/${{ env.DEPLOY_ENV }}/bcda-${{ env.DEPLOY_ENV }}-api-waf-sync-bucket
       - name: Upload and reload
         run: |
           aws s3 cp --no-progress function.zip \
             s3://${{ env.S3_BUCKET }}/function-${{ github.sha }}.zip
```

Comment:
Do the references to the s3 bucket need to be $S3_BUCKET instead of ${{ env.S3_BUCKET }} ?
Not 100 % sure that's relevant, but that is a difference between this workflow and the working ones on bcda-app.